aws-rds/aws-rds.json

```diff
@@ -1801,7 +1758,7 @@
             "filter": ""
           },
           "matchExact": true,
-          "metricName": "ReplicaLag",
```

Comment:
I would prefer to keep original metric here. TransactionLogsDiskUsage can be additional panel, query.

Reply:
There's another panel with the original metric called `OldestReplicationSlotLag/ReplicaLag` this one's a duplicate panel which has title `ReplicationSlotDiskUsage/TransactionLogsDiskUsage`